services/search/pkg/engine/bleve.go

```diff
@@ -32,10 +34,16 @@ import (
 	searchQuery "github.com/opencloud-eu/opencloud/services/search/pkg/query"
 )
 
+const _batchSize = 500
```

Comment:
Do we really want to hard-code the batch-size? This might make sense to have a config var for that, depending on the systems usage (file-sizes etc.).